### DISCUSSION TITLE
Wonderful development - Github and ORCID sign a memorandum of understanding

### DISCUSSION THREAD

**Discussion by abotzki** [LiaScript/LiaScript#169] [General]
https://www.linkedin.com/feed/update/urn:li:activity:7179113860954492928/
A lot of people will find this also very relevant for training material hosted on github ;-)

**Comment by andre-dietrich** [comment]
Sorry for the late response, this is indeed a great information... In two weeks I will be joining of a project in which we try to organize and search OER with semantic web technologies ... and I want to include of this into LiaScript ... Maybe we can have a chat and you can give us some hints ;-)

Have a nice weekend ... 

By the way we, in order to see who and where LiaScript-OER is created, we have created a map on the new website ...

https://liascript.github.io/world-map/

If you want to add your resources, feel free to add your information to our map ;-)
